ramp-engine/rampbkd/model.py

```diff
@@ -20,7 +20,7 @@
 from sqlalchemy.ext.hybrid import hybrid_property
 from sqlalchemy.ext.declarative import declarative_base
 
-from . import config
+from .base import get_deployment_path
```

Comment:
we can maybe just remove this whole file? (or keep that for later?)

Reply:
In this PR I did not touch the new directories. The one I was thinking of removing is rather `databoard/model.py` since there are no more calls to it. But let's keep them now and do a nice cleaning in an upcoming PR.